### ISSUE TITLE
HW1 Flags Spacing

### ISSUE THREAD

**Issue by donei003** [mikeizbicki/ucr-cs100#1442]
Hello,

For the ls assignment, do we need to be able to handle "ls    -     a" or something similiar to that? If there are spaces in the parameters when ls is called, it will create a new index in argv.

**Comment by mikeizbicki** [comment]
I'm not sure exactly what your question is.  Can you rephrase it?

You need to work exactly like GNU ls does for the three flags.

**Comment by donei003** [comment]
Let's say the user types in "ls       -                 a" , there will be three indexes in argv, "ls", "-" and "a". Do we need to be able to handle this case were argv[1] is the first half of the flag and argv[2] is the second half?

**Comment by mikeizbicki** [comment]
How does GNU ls handle that case?  You should do the same thing.
